### PR TITLE
Show the maximum available CPU/Memory on TiFlash Grafana panel

### LINKED ISSUE #3821: Show the maximum available CPU/Memory on TiFlash Grafana panel
## Feature Request

**Is your feature request related to a problem? Please describe:**

Now we can not know what's the maximum available CPU/Memory on Grafana panel, especially on k8s deployed env.

**Describe the feature you'd like:**

Show the maximum available CPU/Memory on TiFlash Grafana panel.
I've created a basic branch for getting the maximum available CPU/Memory quota: https://github.com/JaySon-Huang/tics/commits/fix_cpu_count

The show style could be like this:
image

**Describe alternatives you've considered:**

**Teachability, Documentation, Adoption, Migration Strategy:**

Useful for checking whether TiFlash resource is running out, especially on k8s env

### PR DESCRIPTION
**Pull request by jiaqizho**
### What problem does this PR solve?

Issue Number: close #3821

Problem Summary:

### What is changed and how it works?

todo : need test with `cgroup`

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

File: metrics/grafana/tiflash_summary.json [hunk 1]
```diff
@@ -1028,6 +1028,140 @@
...
+              "step": 10
+            },
+            {
+              "hide": false,
+              "refId": "B"
+            }
+          ],
+          "timeFrom": null,
+          "timeShift": null,
+          "title": "Main Memory Amount",
```
Comment (JaySon-Huang) [inline metrics/grafana/tiflash_summary.json:1163]: Use "Memory Capacity"

File: metrics/grafana/tiflash_summary.json [hunk 2]
```diff
@@ -1028,6 +1028,140 @@
...
+              "expr": "sum(tiflash_system_current_metric_CPUCores{tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\"}) by (instance)",
+              "format": "time_series",
+              "interval": "",
+              "intervalFactor": 2,
+              "legendFormat": "{{instance}}",
+              "refId": "A",
+              "step": 10
+            }
+          ],
+          "title": "Cpu counts",
```
Comment (JaySon-Huang) [inline metrics/grafana/tiflash_summary.json:1093]: "Physical CPU Cores"

**Review by JaySon-Huang — COMMENTED** [review]
Sorry I found I get it wrong before. We need to show the CPU cores and memory limit for each instance, instead of showing the total amount of all instances.

Please refine the two panels you added as the "Memory" panel show.

image

**Comment by Lloyd-Pottiger** [comment]
I will update the value after #4880 merged and support cgroup v2 memory limitation later.

**Pull request closed without merging**